src/core_modules/capture-core/components/WidgetEventSchedule/WidgetEventSchedule.component.js

```diff
@@ -75,70 +48,95 @@ const WidgetEventSchedulePlain = ({
     assignee,
     categoryOptionsError,
     ...passOnProps
-}: Props) => (
-    <div className={classes.wrapper}>
-        <DataSection
-            dataTest="schedule-section"
-            sectionName={i18n.t('Schedule info')}
+}: Props) => {
+    const onSelectOrgUnit = (e: { id: string, displayName: string, path: string }) => {
+        setScheduledOrgUnit({
+            id: e.id,
+            name: e.displayName,
+            path: e.path,
+        });
+    };
+
+    const onDeselectOrgUnit = () => {
+        setScheduledOrgUnit(undefined);
+    };
+
+    useEffect(() => {
+        const formIsValid = () => Boolean(isValidOrgUnit(orgUnit) && scheduleDate);
+        setIsFormValid(formIsValid());
+    }, [orgUnit, scheduleDate, setIsFormValid]);
+
+    return (
+        <Widget
+            noncollapsible
+            borderless
+            header={
+                <></>
+            }
```

Comment:
The `Widget` works fine without passing a header, it's just `flow` that's currently complaining. Can you update the type to `header?:Node` in [widgetCollapsible.types.js](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/Widget/widgetCollapsible.types.js#L5) and [widgetNonCollapsible.types.js](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/Widget/widgetNonCollapsible.types.js#L5) and remove these lines? Thanks!